### ISSUE TITLE
Display highway=give_way and highway=stop

### ISSUE THREAD

**Issue by aceman444** [osmandapp/Osmand#7351]
Display highway=give_way and highway=stop in some rendering profile. The tags on the nodes are very useful for navigation (they give penalty) thus mapping them is beneficial. It would speed up collecting them if the already existing ones in the OSM data would be displayed in OsmAnd.

**Comment by scaidermern** [comment]
highway=stop is already displayed, highway=give_way isn't yet.

**Comment by aceman444** [comment]
Is stop displayed on the map wherever you pan to, or do you mean the announcement when you have a route calculated and the navigation is enabled? That is something completely different.

**Comment by scaidermern** [comment]
They are displayed while driving, not when panning the map. Always rendering them would clutter the map without any real benefit in my opinion.

**Comment by aceman444** [comment]
When you are not driving but walking around the streets and collecting OSM data (which is also a great part of OsmAnd features and usecases) you need to see as much data from OSM as possible to avoid collecting IRL data that is already mapped.
I did not say the stop and give way signs have to be always displayed, but there should be a rendering mode or toggle to show them. Maybe in a rendering mode tailored for cars (e.g. LightRS), they would be useful also while driving.

**Comment by goldfndr** [comment]
Using a [rendering](https://github.com/osmandapp/OsmAnd-resources/tree/master/rendering_styles) template (like *standalone-template*) that *depends* on another, one can add the following (or similar) within the point section for readily viewing these.

```
<point>
  <switch minzoom="14" OSMMapperAssistant="true" moreDetailed="true" iconOrder="40" intersectionSizeFactor="0">
    <case tag="highway" value="stop" icon="stop"/>
    <case tag="highway" value="give_way" icon="give_way" shield="gray_round_shield"/>
  </switch>
</point>
```

**Comment by Dimitar5555** [comment]
They can be added as a separate option in Configure map -> Details view. That way anyone who wants to see them will be able see them.

**Comment by aceman444** [comment]
Exactly. There are already toggleable options for "more detail", "road surfaces", "access restrictions", etc. so these car-oriented features could be added too. This would be very beneficial for OSM mappers that collect this information on the spot so they can see where they already are mapped on the roads.

**Comment by aceman444** [comment]
> Using a [rendering](https://github.com/osmandapp/OsmAnd-resources/tree/master/rendering_styles) template (like _standalone-template_) that _depends_ on another, one can add the following (or similar) within the point section for readily viewing these.
> 
> ```
> <point>
>   <switch minzoom="14" OSMMapperAssistant="true" moreDetailed="true" iconOrder="40" intersectionSizeFactor="0">
>     <case tag="highway" value="stop" icon="stop"/>
>     <case tag="highway" value="give_way" icon="give_way"/>
>   </switch>
> </point>
> ```

Thank you, this actually worked perfectly! I could make a patch with this into [osmassistant.addon.render.xml](https://github.com/osmandapp/OsmAnd-resources/blob/master/rendering_styles/osmassistant.addon.render.xml) .
